contracts/PriceFeed.sol

```diff
@@ -32,25 +32,24 @@ contract PriceFeed is IPriceFeed, PriceModel, Ownable {
 
     function setPrices(PriceData calldata priceData, bytes calldata signature) external {
         address signer = priceVerifier.recoverDataSigner(priceData, signature);
+        uint256 blockTimestampMillseconds = block.timestamp * 1000;
+
         require(isSigner(signer), "Unauthorized price data signer");
-        console.log("Signer OK");
-        console.log("BT: ", block.timestamp * 1000);
-        console.log("PT: ", priceData.timestamp);        
-        require(block.timestamp * 1000 > priceData.timestamp - 15000, "Price data timestamp cannot be from the future");
-        require(block.timestamp * 1000 < priceData.timestamp || block.timestamp * 1000 - priceData.timestamp < maxPriceDelay * 1000, "Price data timestamp too old");
-        console.log("Timestamp OK");
+        // TODO: check the problem with prices on Kovan
+        require(blockTimestampMillseconds > priceData.timestamp - 15000, "Price data timestamp cannot be from the future");
```

Comment:
Let's extract 15000 to a const,
if the problem on Kovan an mainnet perrsist we could have it as a config parameter